purchase_reception_status/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Purchase Reception Status',
+    'version': '12.0.1.0.0',
+    'category': 'Purchases',
+    'license': 'AGPL-3',
+    'summary': 'Add reception status on purchase orders',
+    'author': 'Akretion,Odoo Community Association (OCA)',
+    'maintainers': ['alexis-via'],
+    'website': 'https://github.com/OCA/purchase-workflow',
+    'depends': ['purchase'],
```

Comment:
Add 'stock' too, because if not it doesn't make sense

Reply:
I disagree. It makes sense event without stock. When you purchase a service, you may want to follow the "reception" of this service by manually entering the "received qty", then the field "reception_status" is still usefull.

Reply:
humm okay